api/envoy/extensions/filters/http/bandwidth_limit/v3alpha/bandwidth_limit.proto

```diff
@@ -67,4 +67,12 @@ message BandwidthLimit {
   // Runtime flag that controls whether the filter is enabled or not. If not specified, defaults
   // to enabled.
   config.core.v3.RuntimeFeatureFlag runtime_enabled = 5;
+
+  // Optional the prefix for the response trailers of bandwidth decode/encode delays for stream transfer.
+  // If not set, use the default value "bandwidth-request-delay-ms" and "bandwidth-response-delay-ms".
+  // If set, the trailer name will be set as:
+  // request: response_trailer_prefix + "-bandwidth-request-delay-ms"
+  // response: response_trailer_prefix + "-bandwidth-response-delay-ms"
+  // If EnableMode is DISABLED or REQUEST or delay time = 0, the trailer will not be set.
+  string response_trailer_prefix = 6;
```

Comment:
Do you want to make these trailers optional, such that an operator can enable or disable it?

Reply:
make sense, added property to enable it.